react-slider.js

```diff
@@ -734,6 +734,9 @@
       var className = this.props.handleClassName + ' ' +
         (this.props.handleClassName + '-' + i) + ' ' +
         (this.state.index === i ? this.props.handleActiveClassName : '');
+      var ariaLabelArray = this.props.ariaLabel && this.props.ariaLabel.constructor === Array
```

Comment:
At best we should use `Array.isArray` but because it's not supported by all browsers we need a polyfill. Can you add it at the top where we define our util functions, e.g.

```javascript
var isArray = Array.isArray || function(x) {
  return Object.prototype.toString.call(x) === '[object Array]';
};
```

Reply:
Sure thing.